crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -2160,6 +2160,7 @@ pub enum SuiEvent {
     /// Move-specific event
     #[serde(rename_all = "camelCase")]
     MoveEvent {
+        // TODO: What's the best way to serialize this using `AccountAddress::short_str_lossless` ??
         package_id: ObjectID,
```

Comment:
cc @patrickkuo do you know what's the best way to trim leading zeros for `ObjectID` when serializing this to JSON?

Reply:
Will handle this in a separate PR